src/main/java/com/rackspace/salus/monitor_management/services/MonitorManagement.java

```diff
@@ -1897,8 +1897,8 @@ private void handleReattachedEnvoy(String tenantId, String resourceId, String en
     final List<BoundMonitor> boundMonitors =
         boundMonitorRepository.findAllByTenantIdAndMonitor_IdIn(tenantId, monitorIdsToUnbind);
 
-    log.debug("Unbinding {} from monitorIds={}",
-        boundMonitors, monitorIdsToUnbind);
+    log.debug("Unbinding {} from {} monitors",
```

Comment:
To give some more context to the new log structure
```suggestion
    log.debug("Unbinding boundMonitorCount={} from monitorCount={}",
```